### PR TITLE
feat: prisma config and schemas

### PR DESCRIPTION
**Pull request by RenanRSilva**

### CODE REVIEW

File: package.json
Comment (PedroLuisBrilhadori) [inline package.json:1]: we can create `db:migrate` in `package.json` scripts

example: 
```json
   "db:migrate": "npx prisma migrate"
```

File: package-lock.json
Comment (PedroLuisBrilhadori) [inline package-lock.json:1]: We are using yarn, this file is necessary?

Comment (RenanRSilva) [reply package-lock.json:1]: I think we can delete this file

File: package.json
Comment (RenanRSilva) [reply package.json:1]: Sure, this can be an improvement

**Pull request merged**